<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasureAreaDlg</class>
 <widget class="QDialog" name="MeasureAreaDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>208</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>296</width>
    <height>208</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MeasureArea</string>
  </property>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lineAreaTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>3</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="text">
    <string>Area:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineArea">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>3</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxSimulation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Simulation</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Move Direction</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMoveDirection">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>21</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>51</y>
      <width>65</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Move Velocity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMoveVelocity">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>51</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>81</y>
      <width>64</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Grow Velocity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditGrowVelocity">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>81</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dialMoveDirection">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="maximum">
     <number>359</number>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchTarget">
     <double>45.000000000000000</double>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>10</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>N</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>107</y>
      <width>261</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>label</zorder>
   <zorder>lineEditMoveDirection</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEditMoveVelocity</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEditGrowVelocity</zorder>
   <zorder>dialMoveDirection</zorder>
   <zorder>label_4</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>lineAreaTitle</zorder>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
